PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java

```diff
@@ -242,6 +242,19 @@ public void onClick(View v) {
                 getmDbHandler().setAvatarFace(face);
                 getmDbHandler().setAvatarHair(hair);
                 getmDbHandler().setAvatarCloth(cloth);
+                getmDbHandler().setAvatarBag(0);
+                getmDbHandler().setAvatarGlasses(0);
+                getmDbHandler().setAvatarHat(0);
+                getmDbHandler().setAvatarNecklace(0);
+                getmDbHandler().updateComplete();//set all the complete scenario fields back to 0
+<<<<<<< HEAD
```

Comment:
Please remove git merge comments as they would not let the code compile. Remove similar statements from the rest of the code too.